### PR TITLE
Add hopper to Posts and display proper title (which could be better).

### PR DESCRIPTION
**Pull request by sporifolous**
I don't know if you want this right now, but here it is. Also, would there be a better way to decide what the title should be? I feel like the case switch is too spaghetti.

### CODE REVIEW

File: edit-hopper.php [hunk 1]
```diff
@@ -35,9 +35,19 @@ public function __construct() {
...
+                                break;
+                            case "page":
+                                $box_title = "Edit Pages";
+                            default:
+                                break;
+                        }
 			add_meta_box(
 		        'edit_hopper_box',
-		        __( 'Edit Pages', 'edit_hopper_textdomain' ),
+		        __( $box_title, 'edit_hopper_textdomain' ),
```
Comment (Dilden) [inline edit-hopper.php:50]: I'm pretty sure we can just use the variable **$screen** right here and then skip the entire case switch.

Comment (sporifolous) [inline edit-hopper.php:50]: I can make that work. Something like  
`'Edit '.$screen.'s'`?

Comment (Dilden) [inline edit-hopper.php:50]: Right. It should work just fine for pages and posts. The only thing I can see being a problem, is later on down the road when we add custom post types and options to show it on those as well. Those could be created with dashes in the name which would show up in the meta box title. We could probably query WP, and find the title of the custom post type and drop that in instead.

Comment (sporifolous) [inline edit-hopper.php:50]: That sounds like a much better idea.

**Pull request merged**